waku/v2/protocol/waku_rln_relay/waku_rln_relay_utils.nim

```diff
@@ -181,14 +169,7 @@ proc appendLength*(input: openArray[byte]): seq[byte] =
     output = concat(@len, @input)
   return output
 
-proc toBuffer*(x: openArray[byte]): Buffer =
```

Comment:
Why don't we need this anymore?

Reply:
In order to use it in createRLNInstance, I had to move it at the beginning of the file https://github.com/status-im/nwaku/blob/765e8c5fa6457beb0b878da18a98682c935ac9ac/waku/v2/protocol/waku_rln_relay/waku_rln_relay_utils.nim#L30